src/shared/components/ResourceDetails/ResourceDetails.js

```diff
@@ -389,6 +395,11 @@ function Resource({
   const resourceDetailsCard = (
     <ResourceDetailsCard
       title={title ?? t('common.headers.resource-details')}
+      wrapperClassname={
+        isColumnLayoutEnabled && layoutColumn.layout === 'MidColumnFullScreen'
+          ? 'resource-overview__details-wrapper'
+          : null
+      }
```

Comment:
You don't need to check if column layout is enabled, we want to have displayed it like this in every case.
```suggestion
      wrapperClassname={'resource-overview__details-wrapper'}
```